tests/test_symbols.py

```diff
@@ -56,17 +56,19 @@ def assertLessEqual(self, *args, **kwargs): ...
     def assertTrue(self, *args, **kwargs): ...
 
     def test_equality(self):
+        DEFINITELY = 2
         for x in self.generate_symbols():
-            self.assertTrue(x.maybe_equals(x))
+            self.assertEqual(DEFINITELY, x.maybe_equals(x))
             self.assertTrue(x.equals(x))
 
         for x, y in zip(self.generate_symbols(), self.generate_symbols()):
-            self.assertTrue(x.maybe_equals(y))
+            self.assertTrue(DEFINITELY, x.maybe_equals(y))
```

Comment:
I know this is already merged but was this supposed to have been updated from `assertTrue`?

Reply:
ah, good catch. I can fix it in a followup.